packages/dvilela/contracts/meme_factory/contract.py

```diff
@@ -185,25 +180,43 @@ def get_summon_data(
         """Get the data from the Summoned event."""
         meme_address = web3.Web3.to_checksum_address(meme_address)
         contract_instance = cls.get_instance(ledger_api, contract_address)
+        meme_summons = getattr(contract_instance.functions, "memeTokenNonces")  # noqa
+        token_nonce = meme_summons(meme_address).call()
+        return {
+            "token_nonce": token_nonce,
+        }
```

Comment:
where are you planning to use this? token nonce to meme address mapping only exists past the point of unleash

Reply:
Yes, I knew it only exists after unleash, and still there is an error in the logic indeed. Fixing it now.